tools/sim/README.md

```diff
@@ -10,31 +10,30 @@ git clone https://github.com/commaai/openpilot.git
 # Add export PYTHONPATH=$HOME/openpilot to your bashrc
 # Have a working tensorflow+keras in python3.7.3 (with [packages] in openpilot/Pipfile)
 ```
-## Install (in tab 1)
+## Install (in tab or terminal 1)
```

Comment:
These are pretty much synonyms in this context. Can you just pick one and fix the other references?

Reply:
@adeebshihadeh Pull request updated.